examples/generic_connector_configs/README.md

```diff
@@ -29,9 +30,47 @@ target’s CA to Secretless’ trusted certificate pool.
 > Replace the service prefix `service#` with an appropriate service
 > or use a different provider as needed.
 
+<<<<<<< HEAD
 > **Protip:** Your target should be either `http://api-target.com` or `api-target.com`.
 A URL that starts with https will not work.
+=======
 ___
+
+### Docker Registry API
+
+This example can be used to interact with [Docker's V2 Registry API](https://docs.docker.com/registry/spec/api/#overview).
+
+The configuration file for the Docker Registry API can be found at
+[docker_registry_secretless.yml](./docker_registry_secretless.yml).
+
+#### How to use this connector
+* Edit the supplied configuration to get your Docker Registry
+[Token](https://docs.docker.com/registry/spec/auth/jwt/)
+* Run Secretless with the supplied configuration(s)
+* Query the Docker Registry API using `http_proxy=localhost:8021 curl <Registry Endpoint URL>/{Request}`
+
+#### Example Usage
+<details>
+  <summary><b>How to use this connector locally</b></summary>
+  <ol>
+    <li>Set up a <a href="https://docs.docker.com/registry/deploying/">
+    local Registry</a> or use one from <a href="https://hub.docker.com">Dockerhub</a></li>
+    <li>Make a request to the Registry API to get the <a href="https://docs.docker.com/registry/spec/auth/oauth/">
+    OAuth2 Token</a>.</li>
+    <li>Run Secretless locally</li>
```

Comment:
You should add a step to add a token to the local keychain. If you pull master, you can copy the exact language/HTML from the other config connectors.